qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -391,3 +440,20 @@ def _compose_transforms(basis_transforms, source_basis, source_dag):
                 )
 
     return mapped_instrs
+
+
+def _compute_incomplete_basis(target):
+    incomplete_basis_gates = []
+    size_dict = defaultdict(lambda: 0)
+    size_dict[1] = target.num_qubits
```

Comment:
Which single-qubit instruction are we assuming exists here?

Reply:
This is any single qubit instruction which doesn't work on every qubit. I'm not sure there will be any, but if there are any single qubit instructions for a subset of qubits we want to know that.

Reply:
Ah ok, I understand the point of the `continue` within the loop now as well.